File: warehouse-system/src/main/java/com/who/warehousesystem/dto/WorkerSaveDto.java
package com.who.warehousesystem.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class WorkerSaveDto {

    private String enName;
    private String arName;
    private String mobile;
}

File: WarehouseSystem/WarehouseSystem/Dto/WorkerSaveDto.cs
using Newtonsoft.Json;

namespace WarehouseSystem
{
    class WorkerSaveDto
    {
        public string enName { get; set; }
        public string arName { get; set; }
        public string mobile { get; set; }

        public WorkerSaveDto (string enName, string arName, string mobile)
        {
            this.enName = enName;
            this.arName = arName;
            this.mobile = mobile;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
